### ISSUE TITLE
Type ******* is missing a "__resolveType" resolver.

### ISSUE THREAD

**Issue by proutek** [dotansimha/graphql-code-generator#3225]
**Describe the bug**
Since 1.10 I'm getting warning when generating types from schema with interface.
Type ******* is missing a "__resolveType" resolver. Pass false into "resolverValidationOptions.requireResolversForResolveType" to disable this warning.

Generated definitions seems to be fine.

**To Reproduce**
Just check this codesandbox using schema from project homepage https://graphql-code-generator.com/

https://codesandbox.io/s/unruffled-kapitsa-tn66e

**Output on command line**
```
Type "Node" is missing a "__resolveType" resolver. Pass false into "resolverValidationOptions.requireResolversForResolveType" to disable this warning.
Type "SearchResult" is missing a "__resolveType" resolver. Pass false into "resolverValidationOptions.requireResolversForResolveType" to disable this warning.
  ✔ Parse configuration
  ✔ Generate outputs
Done in 0.94s.
```

**Comment by ardatan** [comment]
It's fixed in #3213
You can try the canary version;
`1.10.1-alpha-7abaacf0.6`

**Comment by proutek** [comment]
It's is. Thank You! I can confirm the warning is gone.
Sorry for redundant issue.

**Comment by dotansimha** [comment]
Fixed in 1.11.0
